backend/backend/websocket_routes.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import APIRouter, WebSocket
+from .speech_client import client
+import asyncio
+import logging
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
I've added a logging config to __init__.py at the top level so this line isn't needed anymore